File: video-studio/scripts/contact-small-shower-ui.mjs
import fs from 'node:fs/promises';import sharp from '../../node_modules/sharp/lib/index.js';
const names=['chat','card','details','share','rsvp','confirmed','calendar','directions'];
const tiles=await Promise.all(names.map(async(n,i)=>({input:await sharp('public/projects/small-shower/demo-'+n+'.png').resize(323,600).jpeg({quality:78}).toBuffer(),left:(i%4)*323,top:Math.floor(i/4)*600})));
await sharp({create:{width:1292,height:1200,channels:3,background:'#ffffff'}}).composite(tiles).jpeg({quality:80}).toFile('out/small-shower/ui-contact.jpg');

File: video-studio/scripts/contact-small-shower-ui.py
from PIL import Image,ImageOps,ImageDraw
from pathlib import Path
root=Path.cwd();names=['chat','card','details','share','rsvp','confirmed','calendar','directions'];sheet=Image.new('RGB',(1292,1200),'white')
for i,n in enumerate(names):
 im=Image.open(root/'public/projects/small-shower'/f'demo-{n}.png').convert('RGB').resize((323,600));sheet.paste(im,((i%4)*323,(i//4)*600))
sheet.save(root/'out/small-shower/ui-contact.jpg',quality=80)
